colp_build/sql/name.sql

```diff
@@ -9,104 +9,110 @@ SELECT
-		                                            ),
-	                                            'AVE\)|AVE\.\)',
-	                                            'AVENUE)'
-	                                            ),
-                                            '\(AVE|\(AVE\.',
-                                            '(AVENUE'
-                                            ),
-                                        '\sAVE\s|\sAVE\.\s|\sAVE-\s|\sAVE\.\s|\sAV\s',
-                                        ' AVENUE '
-                                        ),
-                                    'HUNTS%POINT%AVENUE%REHAB',
-                                    'HUNTS POINT AVENUE REHAB'
-                                    ),
-                                'AUTH\s',
-                                'AUTHORITY '
-                                ),
-                            'BKLN|BK|BKLYN',
-                            'BROOKLYN'
-                            ),
-                        '\sBX\s|\sBRNX\s|/BX\s|\sBX\.\s',
-                        ' BRONX '
-                        ),
-                    'QNS',
-                    'QUEENS'
-                    ),
-                '\sSI\s',
-                ' STATEN ISLAND '
-                ),
-            'MN\s|MAHN\s',
-            'MANHATTAN '
-            ),
-    'REHABS\(|REHAB\(',
-    'REHAB ('
-    ),
-'REHABS|REHAS',
-'REHAB'
-)) as new_name,
+			REGEXP_REPLACE(
+				REGEXP_REPLACE(
+					REGEXP_REPLACE(
+						REGEXP_REPLACE(
+							REGEXP_REPLACE(
+								REGEXP_REPLACE(
+									REGEXP_REPLACE(
+										REGEXP_REPLACE(
+											REGEXP_REPLACE(
+												REGEXP_REPLACE(
+													REGEXP_REPLACE(
+														REGEXP_REPLACE(
+															REGEXP_REPLACE(
+																REGEXP_REPLACE(
+																	REGEXP_REPLACE(
+																		REGEXP_REPLACE(
+																			REGEXP_REPLACE(
+																				REGEXP_REPLACE(
+																					REGEXP_REPLACE(
+																						REGEXP_REPLACE(
+																							REGEXP_REPLACE(
+																								REGEXP_REPLACE(
+																									REGEXP_REPLACE(
+																										REGEXP_REPLACE(
+																											REGEXP_REPLACE(' '||TRIM(parcel_name)||' ', 
+																											'\s&\s|&', 
+																											' AND '
+																											), 
+																										'PLAYLOT\s|PLGD\s|PLAYGRD\s|PLGRD\s|PLAYGND\s|PLYGRND/|PLYGRND\s|PLYGD\s|PLYGD|PLAYGRND\s|PLYGRD\s|PLYGRD/|PLGRD/|PLGRD\s|PLGD.\s|PLGD\s|PLYGD/|PLYGD|PLYGD\s|PLAYGD\s|PLGND\s|PLGD/|PLGD-|PLAYGND-', 
+																										'PLAYGROUND '
+																										),
+																									'\@',
+																									'AT'
+																									),
+																								'HSES',
+																								'HOUSES'
+																								),
+																							'\s\(GROUP 1\s',
+																							'\s(GROUP 1)\s'
+																							),
+																						'\sPK\s|\sPK-',
```

Comment:
Note: Is there any risk of this accidently replacing Pre-K?  Check in QAQC review.